Firestore/Example/FuzzTests/FuzzingResources/Serializer/Corpus/ConvertTextToBinary.sh

```diff
@@ -0,0 +1,57 @@
+# Copyright 2018 Google
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Converts text protos to binary protos and writes the output binary protos to
+# the output folder that is specified in SCRIPT_OUTPUT_FILE_0, which is
```

Comment:
Can you add a comment on where `SCRIPT_OUTPUT_FILE_0` is defined?

Reply:
There is already an explanation about it "retrieved from the Output Files of the Run Script Build Phase that executes this script", but I made it more clear (I guess) and added explanation of `SCRIPT_INPUT_FILE_0` as well.

Reply:
There was a comment explaining how it is retrieved from the Build Phase but I made it more clear and also added documentation about `SCRIPT_INPUT_FILE_0`